### ISSUE TITLE
Add possibility to load dataset in the config

### ISSUE THREAD

**Issue by fdtomasi** [slipguru/palladio#16]
For example, if I have a dataset in a non-standard format, there is no clear way on how I can load my dataset in palladio.
Also, if for example I want to use palladio on a dummy dataset, maybe `sklearn.datasets.makeclassifier`, I need to load my data and labels in the config.

**Comment by matteobarbieri** [comment]
Agreed, all processes load it anyway.
We could probably seize the opportunity and make it compatible with the sklearn.datasets.load_dataset* format, I mean an object which has the `.data` and `.target` (and whatever) properties

**Comment by fdtomasi** [comment]
Fix in #19.

Now data and labels are loaded in the config.
